src/Storage/Device/S3.php

```diff
@@ -168,9 +172,9 @@ public function getPath(string $filename): string
      *
      * @throws \Exception
      *
-     * @return bool
+     * @return int
      */
-    public function upload($source, $path): bool
+    public function upload($source, $path, $chunk = 1, $chunks = 1): int
```

Comment:
Maybe throw a warning that we don't support chunks with S3 yet. Or is this going to be implemented at a later point?

Reply:
This has to be implemented before we make it work with Appwrite.